README.md

```diff
@@ -163,10 +170,12 @@ initialization scripts (and later daemons) to pick from.
 ovs-vsctl set Open_vSwitch . external_ids:k8s-api-server="$OVN_CENTRAL_IP:8080"
 ```
 
-Install the following additional packages for python.
+Clone this repository and install the executables.
```

Comment:
Sure.. I was just being pedant and note that pip install actually installs all the python modules in /usr/local/lib/python2.7/dist-packages/ovn-kubernetes and creates entry points for the executables.

You know that I warned you about me being pedant and annoying in reviews!!!